webapp/components/new-test-results-history-grid.js

```diff
@@ -88,21 +88,22 @@ class TestResultsGrid extends PolymerElement {
 
   static get observers() {
     return [
-      'displayCharts(showTestHistory)'
+      'displayCharts(showTestHistory, path)',
     ];
   }
 
   static get is() {
     return 'new-test-results-history-grid';
   }
 
-  displayCharts(showTestHistory) {
-    if (!showTestHistory || this.historicalData !== undefined) {
+  displayCharts(showTestHistory, path) {
+    if (!path || !showTestHistory || path.split('?')[0].slice(-5) !== ".html") {
```

Comment:
@KyleJu I believe this is now accurately checking for subtests urls. Can you let me know if there are any edge cases you can think of that I'm not covering?

Reply:
Could you use this method for checking instead? https://github.com/web-platform-tests/wpt.fyi/blob/7d1370ea05466fc84533ebed782d348756e102d5/webapp/components/path.js#L70

Reply:
Done 👍